src/server_manager/web_app/ui_components/outline-server-view.js

```diff
@@ -850,6 +878,33 @@ export class ServerView extends DirMixin(PolymerElement) {
     this.dispatchEvent(makePublicEvent('RemoveAccessKeyRequested', {accessKeyId: accessKey.id}));
   }
 
+  setServerTransferredData(totalBytes) {
+    this.totalInboundBytes = totalBytes;
+  }
+
+  updateAccessKeyRow(accessKeyId, fields) {
+    let newAccessKeyRow;
+    if (accessKeyId === MY_CONNECTION_USER_ID) {
+      newAccessKeyRow = Object.assign({}, this.get('myConnection'), fields);
+      this.set('myConnection', newAccessKeyRow);
+    }
+    for (let ui in this.accessKeyRows) {
+      if (this.accessKeyRows[ui].id === accessKeyId) {
```

Comment:
I don't understand what this `[ui]` is doing.

Reply:
Renamed in 5f7f838880c3e8f205919c12ad33761d7e499b4c.  It's the index of the key, since we're using `let in`.  This allows us to set the access key row `dom-repeat` instances with `this.set()`